sig-apps/annual-report-2022.md

```diff
@@ -4,81 +4,73 @@
 
 1. What work did the SIG do this year that should be highlighted?
 
-   -
-   -
-   -
+   - We're working closely with [WG Batch](https://git.k8s.io/community/wg-batch/README.md) towards improving batch workloads for use in HPC, AI/ML and data analytics workflows.
+   - [Mentoring cohort](https://github.com/kubernetes/community/issues/6665) which allowed us to grow our reviewers' list.
 
 2. What initiatives are you working on that aren't being tracked in KEPs?
 
-   -
-   -
-   -
-
-
+   - None.
 
 3. KEP work in 2022 (v1.24, v1.25, v1.26):
   - alpha:
     - [2804 - Consolidate Workload controllers life cycle status](https://github.com/kubernetes/enhancements/tree/master/keps/sig-apps/2804-consolidate-workload-controllers-status) - v1.24
     - [961 - Implement maxUnavailable for StatefulSets](https://github.com/kubernetes/enhancements/tree/master/keps/sig-apps/961-maxunavailable-for-statefulset) - v1.24
+    - [3329 - Retriable and non-retriable Pod failures for Jobs](https://github.com/kubernetes/enhancements/tree/master/keps/sig-apps/3329-retriable-and-non-retriable-failures) - v1.25
+    - [3017 - Pod Healthy Policy for PDB](https://github.com/kubernetes/enhancements/tree/master/keps/sig-apps/3017-pod-healthy-policy-for-pdb) - v1.26
   - beta:
-    - [2307 - Job tracking without lingering Pods](https://github.com/kubernetes/enhancements/tree/master/keps/sig-apps/2307-job-tracking-without-lingering-pods) - v1.26
     - [2879 - Track ready Pods in Job status](https://github.com/kubernetes/enhancements/tree/master/keps/sig-apps/2879-ready-pods-job-status) - v1.24
+    - [3140 - TimeZone support in CronJob](https://github.com/kubernetes/enhancements/tree/master/keps/sig-apps/3140-TimeZone-support-in-CronJob) - v1.25
+    - [2307 - Job tracking without lingering Pods](https://github.com/kubernetes/enhancements/tree/master/keps/sig-apps/2307-job-tracking-without-lingering-pods) - v1.26
+    - [3329 - Retriable and non-retriable Pod failures for Jobs](https://github.com/kubernetes/enhancements/tree/master/keps/sig-apps/3329-retriable-and-non-retriable-failures) - v1.26
   - stable:
     - [2214 - Indexed Job](https://github.com/kubernetes/enhancements/tree/master/keps/sig-apps/2214-indexed-job) - v1.24
     - [2232 - Suspend Job](https://github.com/kubernetes/enhancements/tree/master/keps/sig-apps/2232-suspend-jobs) - v1.24
+    - [1591 - Allow DaemonSets to surge during update like Deployments](https://github.com/kubernetes/enhancements/tree/master/keps/sig-apps/1591-daemonset-surge/kep.yaml) - v1.25
     - [2599 - minReadySeconds for StatefulSets](https://github.com/kubernetes/enhancements/tree/master/keps/sig-apps/2599-minreadyseconds-for-statefulsets) - v1.25
 
-
 ## Project health
 
 1. What areas and/or subprojects does your group need the most help with?
    Any areas with 2 or fewer OWNERs? (link to more details)
 
-   -
-   -
-   -
+   - The mentoring cohort we've run in 2022 attracted a few new members, but we still need more.
```

Comment:
Is there an issue/repo that would be good to direct folk to? these get consolidated and shared out with the GB / end users / member companies, having a place to direct potential folks would be good to include

Reply:
I've added link to the previous one, but I need to talk to several folks during KubeCon next week how to solve the limitations I've run into with this.